Userland/Libraries/LibELF/Image.cpp

```diff
@@ -169,7 +169,11 @@ StringView Image::table_string(unsigned table_index, unsigned offset) const
             dbgln("SHENANIGANS! Image::table_string() computed offset outside image.");
         return {};
     }
+#if defined(AK_OS_SERENITY)
```

Comment:
1. Why is this needed now? This code isn't new as far as I can tell. (Maybe it's used for prekernel things now?)
2. Why do we want final page size instead of current page size here? If it's for prekernel things, I'm guessing it's because we're cross-compiling in some sense (?)

Implementation-wise, it'd probably nice to have a `SERENITY_PAGE_SIZE` define somewhere that we could use here so we don't need to duplicate the ifdef in several places, and to make the intent of what's happening clear. And the commit message should say why that's what we want.